flocker/node/agents/blockdevice_manager.py

```diff
@@ -190,3 +283,32 @@ def get_mounts(self):
         return (MountInfo(blockdevice=FilePath(mount.device),
                           mountpoint=FilePath(mount.mountpoint))
                 for mount in mounts)
+
+    def bind_mount(self, source_path, mountpoint):
+        try:
+            check_output([b"mount", "--bind", source_path.path,
+                          mountpoint.path],
+                         stderr=STDOUT)
+        except CalledProcessError as e:
+            raise BindMountError(source_path=source_path,
+                                 mountpoint=mountpoint,
+                                 source_message="\n".join([str(e), e.output]))
```

Comment:
:question: Would it make sense to have some sort of prettify helper for this repeated operation?